src/util/strencodings.cpp

```diff
@@ -201,20 +201,24 @@ std::string DecodeBase64(const std::string& str, bool* pf_invalid)
     return std::string((const char*)vchRet.data(), vchRet.size());
 }
 
-std::string EncodeBase32(Span<const unsigned char> input)
+std::string EncodeBase32(Span<const unsigned char> input, bool pad)
```

Comment:
nit: same with the string one below
```suggestion
std::string EncodeBase32(Span<const unsigned char> input, const bool pad)
```

Reply:
Lets have the conversation in just one place: https://github.com/bitcoin/bitcoin/pull/19845#discussion_r489748399